project/MimaExcludes.scala

```diff
@@ -100,6 +100,30 @@ object MimaExcludes {
           "org.apache.spark.sql.SQLContext.setSession"),
         ProblemFilters.exclude[MissingMethodProblem](
           "org.apache.spark.sql.SQLContext.createSession")
+      ) ++ Seq(
+        // SPARK-10708: Consolidate sort shuffle implementations
```

Comment:
Ah, but it excludes all DeveloperAPI classes. I'm going to go ahead and exclude the shuffle, unsafe, and collection packages.